platform/security/sts/security-sts-attributequerycommon/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
❓ What does this guy do?

Reply:
In general, mockito does not allow final methods to be mocked and stubbed. This (along with the dependency change) upgrades mockito to the version that allows for this to occur.